Modules/_testexternalinspection.c

```diff
@@ -414,7 +414,11 @@ get_py_runtime(pid_t pid)
 static uintptr_t
 get_async_debug(pid_t pid)
```

Comment:
This change is to avoid returning NULL without an error set as things under search_map_for_section may return without setting an error.